soc/raspberrypi/rpi_pico/rp2350/Kconfig.soc

```diff
@@ -15,5 +14,11 @@ config SOC_RP2350A
 	bool
 	select SOC_SERIES_RP2350
 
+config SOC_RP2350A_M33
```

Comment:
this hasn't been fixed

Reply:
"rp2350a_m33" string missing for `config SOC` ?